docs/tutorials/essentials/part-6-performance-normalization.md

```diff
@@ -362,17 +363,18 @@ const notificationsSlice = createSlice({
     // highlight-end
   },
   extraReducers(builder) {
-    builder.addCase(fetchNotifications.fulfilled, (state, action) => {
-      state.push(...action.payload)
-      // highlight-start
-      state.forEach(notification => {
-        // Any notifications we've read are no longer new
-        notification.isNew = !notification.read
-      })
-      // highlight-end
-      // Sort with newest first
-      state.sort((a, b) => b.date.localeCompare(a.date))
-    })
+    builder
+      .addCase(fetchPosts.fulfilled, (state, action) => {
```

Comment:
:hand: This just changed from `fetchNotifications.fulfilled` to `fetchPosts.fulfilled`, which doesn't look right.